python-sdk/nuscenes/eval/detection/tests/test_main.py

```diff
@@ -134,6 +133,7 @@ def test_delta(self):
         # After changing to measure center distance from the ego-vehicle this changed to 0.2199307290627096
         # Changed to 1.0-mini. Cleaned up build script. So new basline at 0.24954451673961747
         self.assertAlmostEqual(metrics['weighted_sum'], 0.24954451673961747)
+        print("Passed unit test test_delta")
```

Comment:
@oscar-nutonomy  I added this print statement as a sanity check.  Let me know if I should remove this (and the one in test_nuscenes.py)